awscli/customizations/s3/factory.py

```diff
@@ -69,11 +101,42 @@ def create_transfer_manager(self, params, runtime_config,
 
     def _compute_transfer_client_type(self, params, runtime_config):
```

Comment:
trivial: there's another function in the codebase named `_create_default_transfer_manager()`. Might want to rename it `_create_classic_transfer_manager()`